styleguide/derek/global-components/filter-bar/filter-accordion.js

```diff
@@ -0,0 +1,64 @@
+// Filter bar plugin
+(($) => {
+  $.fn.rsFilterBarAccordion = function rsFilterBarAccordion() {
+    const $filterBar = $(this);
+    const $window = $(window);
+    const activeClass = 'rsFilter-sideBar-active';
+    const contentClass = '.rsFilter-sideBar-accItemContent';
+    const labelClass = '.rsFilter-sideBar-accLabel';
+    const $item = $filterBar.find(labelClass);
+    const $hamburger = $filterBar.find('.rsFilter-sideBar-hamburger');
+    const $formContent = $filterBar.find('.rsFilter-sideBar-content');
+    const frameRate = 150;
+
+    const actions = {
+      openActive($el) {
+        // close all siblings
+        $.when(this.closeAll($el)).done(() => {
+          // then open content for clicked item
+          const $content = $el.next(contentClass);
+          $content.slideToggle(frameRate, () => {
+            if ($content.is(':visible')) {
+              $el.addClass(activeClass);
+            } else {
+              $el.removeClass(activeClass);
+            }
+          });
+        });
+      },
+      closeAll($el) {
+        // some promise here
+        const $siblings = $el.parent().siblings();
+        $siblings.find(labelClass).removeClass(activeClass);
+        $siblings.find(contentClass).slideUp(frameRate);
+      },
+      toggleForm() {
+        $formContent.slideToggle(frameRate);
+      },
+      resizeTimer: null,
+      resetForm() {
+        clearTimeout(this.resizeTimer);
+        this.resizeTimer = setTimeout(() => {
+          if ($window.outerWidth() > 768 && !$formContent.is(':visible')) {
+            $formContent.show();
+          }
+        }, 250);
```

Comment:
What is the purpose of this timeout here?

Reply:
when you're resizing from mobile to desktop. if you collapsed it in mobile, then move back to desktop, it needs to show up again. The timeout is so that it won't run a million times for every pixel resized

Reply:
can we set the timeout to `100` instead of `250` then? The human eye can perceive up to about 1/10 of a second, so it may help reduce jitter, and still prevent the busy waiting there.